scripts/popup.js

```diff
@@ -1,3 +1,98 @@
+function getFormData(){
+    var formElements = document.getElementById('metadata-form').elements;
+    var formData = {};
+
+    for(var i = 0 ; i < formElements.length ; i++){
```

Comment:
Alternatively, I think you can use `forEach` here like you did below -- this is the best way.